sstables/sstables.cc

```diff
@@ -1394,6 +1394,7 @@ future<> sstable::update_info_for_opened_data(sstable_open_config cfg) {
     if (cfg.load_first_and_last_position_metadata) {
         co_await load_first_and_last_position_in_partition();
     }
+    get_column_translation(*_schema);
```

Comment:
I fail to see how this dropped-returned-value and using-const-schema is doing something here.
If it _is_ important to get the side effects (a potential change in `_state` of `_column_translation`) then `get_column_translation` should be named differently.

Reply:
It's warming up the _column_translation, so that schema-less get_column_translation() is always valid.

I need schema-less get_column_translation(), which means "get any mapping", in places which don't have easy access to latest schema. I don't want to use sstable::_schema in such places because that can be older than current schema used in other places and cause thrashing of _column_translation back and forth.

Reply:
Fine then.
I suggest that you either rename `get_column_translation` to something else (e.g. `prepare_and_get_column_translation`) or add a comment on this specific invocation of this function - because it looks like it is a leftover from previous work.

Reply:
Added